browser/ui/views/tabs/brave_tab_group_header.cc

```diff
@@ -99,8 +120,13 @@ void BraveTabGroupHeader::LayoutTitleChipForVerticalTabs() {
   title_chip_->SetBoundsRect(title_bounds);
 
   // |title_| is a child view of |title_chip_| and there could be |sync_icon_|
-  // before |title_|. So expand |title_|'s width considering that.
-  title_->SetSize({title_bounds.width() - title_->x(), title_->height()});
+  // before |title_| and |plus_icon_| after |title_|. So expand |title_|'s width
+  // considering that.
+  const int title_width = title_bounds.width() - title_->x() - kIconSize;
+
+  title_->SetSize({title_width, title_->height()});
+  plus_icon_->SetBoundsRect(
+      {title_->bounds().right(), title_bounds.y(), kIconSize, kIconSize});
```

Comment:
Hi @thirumurugan-git , sorry but I think I missed one thing. Shouldn't we `plus_icon_->SetVisible(ShouldShowVerticalTabs())` around line 72?